app/controllers/merchant_controller.rb

```diff
@@ -1,9 +1,10 @@
-class MerchController <ApplicationController
+class MerchantController <ApplicationController
+
   def show
     if current_user == nil || current_user.role != "merchant"
       redirect_to "/error404"
     else
-      @merch = User.find_by(id: session[:user_id])&.merchant
+      @merchant = User.find_by(id: session[:user_id])&.merchant
```

Comment:
I started to get confused by merch and comfortable with merchant (singular) because it is used in so many places so I changed. Hope nobody wants to fight about it!

Reply:
![](https://media1.giphy.com/media/8OJdqYqN1Nii3UTD6l/giphy.gif?cid=5a38a5a263257a7525cf9e356576f3088f181375861722b8&rid=giphy.gif)